src/pages/MainPage/components/ProceedContainer/components/SumaryContainer/components/SumaryBody/components/TableTitle/styles.js

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  background: ${(props) => props.theme.colors.ligthGray};
+  padding: 10px;
+  border: 1px solid black;
```

Comment:
takie kolory jak black i white tez warto trzymac w colors. Bywa, ze black nie jest calkiem black, a white nie jest całkiem white :P

a gray na skali szarości, tj. zamiast lightGray to np. gray[800]. Często designerzy uzywaja kilka odcieni i potem trzebaby robic light, mediumLight, ultraLight itd. - niewygodnie i mniej czytelnie

Reply:
1zastosuję 2 pomijam
Done.